src/main/java/com/gmail/goosius/siegewar/playeractions/PlaceBlock.java

```diff
@@ -65,7 +65,8 @@ public static void evaluateSiegeWarPlaceBlockRequest(Player player, Block block,
 			//Standing Banner placement
 			if (Tag.BANNERS.isTagged(mat) && !mat.toString().contains("_W"))
                 try {
-                    evaluatePlaceStandingBanner(player, block);
+					if (isValidPlacement(block))
```

Comment:
It looks to me like this would need an 
```java
		else
		    return;
```
on it to keep the old behviour, no?

Because the older
```java
		//If no townblocks are nearby, do normal block placement	
		if (nearbyCardinalTownBlocks.size() == 0)	
			return;
```
tests are being removed later on.

Reply:
I don't think it's needed, if it's false then it would automatically return since there wouldn't be anything else to run.

Reply:
Ah you're right, I was just looking at the diff on GH and wasn't paying attention to what method it was in.